src/controllers/trigger/common/trigger.js

```diff
@@ -65,6 +65,10 @@ export async function triggerFromData(context, config, auditContext = {}) {
         return !auditConfig.getAuditTypeConfig(auditType)?.disabled();
       });
 
+      if (!sitesToAuditForType.length) {
+        message.push(`No site is not enabled for ${auditType} audit type`);
```

Comment:
I think it's better to add an early return here in order not to call `sendAuditMessages` at all - which will log the same message with `undefined`

Alternatively, the empty check could be added in the `sendAuditMessages` function instead of here